scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,945 @@
+
+    Returns:
+        pd.DataFrame: DataFrame with New Columns
+    """
+    df[tgt_col] = df[src_col].str.split("_").str[-1]
+    df[tgt_col] = df[tgt_col].str.replace(r"^(?!computed).*",
+                                          "dcs:CensusPEPSurvey",
+                                          regex=True)
+    df[tgt_col] = df[tgt_col].str.replace('computed',
+                                          "dcs:dcAggregate/CensusPEPSurvey",
+                                          regex=False)
+    df[src_col] = df[src_col].str.replace("_computed", "")
+    return df
+
+
+def _load_df(path: str,
+             file_format: str,
+             header: str = None,
+             skip_rows: int = None,
+             encoding: str = "UTF-8") -> pd.DataFrame:
+    """
+    Returns the DataFrame using input path and config
+    Args:
+        path (str): Input File Path
+        file_format (str): Input File Format
+        header (str, optional): Input File Header Index. Defaults to None.
+        skip_rows (int, optional): Skip Rows Value. Defaults to None.
+        encoding (str, optional): Input File Encoding. Defaults to None.
+
+    Returns:
+        df (pd.DataFrame): Dataframe of input file
+    """
+    df = None
+    if file_format.lower() == "csv":
+        df = pd.read_csv(path, header=header, encoding=encoding)
+    elif file_format.lower() == "txt":
+        df = pd.read_table(path,
+                           index_col=False,
+                           delim_whitespace=True,
+                           engine='python',
+                           header=header,
+                           skiprows=skip_rows,
+                           encoding=encoding)
+    elif file_format.lower() in ["xls", "xlsx"]:
+        df = pd.read_excel(path, header=header)
+    return df
+
+
+def _merge_df(df_1: pd.DataFrame, df_2: pd.DataFrame) -> pd.DataFrame:
+    if df_1 is None:
+        return df_2
+    return pd.concat([df_1, df_2], axis=0)
+
+
+def _transpose_df(df: pd.DataFrame,
+                  common_col: list,
+                  data_cols: list,
+                  default_col="SV") -> pd.DataFrame:
+    """
+    Transpose the data_cols into single column named as default_col
+    and concatinates the default_col with common_cols
+    Args:
+        df (pd.DataFrame): Dataframe with cleaned data
+        common_col (list): Dataframe Column list
+        data_cols (list): Dataframe Column list
+
+    Returns:
+        pd.DataFrame: Dataframe
+    """
+    res_df = pd.DataFrame()
+    for col in data_cols:
+        cols = common_col + [col]
+        tmp_df = df[cols]
+        tmp_df.columns = common_col + ["Count_Person"]
+        tmp_df[default_col] = col
+        res_df = pd.concat([res_df, tmp_df])
+    return res_df
+
+
+def _create_sv(desc: str, age: str) -> str:
+    if age == 100:
+        return f"Count_Person_{age}OrMoreYears_{desc}"
+    if age == 85:
+        return f"Count_Person_{age}OrMoreYears_{desc}"
```

Comment:
Doesn't look like we need separate conditionals for these two. Just use:

if age == 85 or age == 100: